pyproject.toml

```diff
@@ -23,6 +23,8 @@ python = "^3.10.0"
 pygithub = "^1.58"
 flask = "^2.3"
 flask-httpauth = "^4.8"
+jsonschema = "^4.17"
```

Comment:
Reminder to wait for the runner charm to pin the version of this repo to a specific commit before merging, otherwise the charm will start failing due to missing dependencies

Reply:
Actually, it turns out that the installation from git includes the required dependencies, so no need to wait

Reply:
```
python3 -m pip install git+https://github.com/canonical/repo-policy-compliance@ce8555c56b2ae28235996714849a60459515f6fc
Collecting git+https://github.com/canonical/repo-policy-compliance@ce8555c56b2ae28235996714849a60459515f6fc
  Cloning https://github.com/canonical/repo-policy-compliance (to revision ce8555c56b2ae28235996714849a60459515f6fc) to /private/var/folders/w7/lb2jf0814tz5sdz8mcq6dywh0000gn/T/pip-req-build-n__ds2mi
  Running command git clone --filter=blob:none --quiet https://github.com/canonical/repo-policy-compliance /private/var/folders/w7/lb2jf0814tz5sdz8mcq6dywh0000gn/T/pip-req-build-n__ds2mi
  Running command git rev-parse -q --verify 'sha^ce8555c56b2ae28235996714849a60459515f6fc'
  Running command git fetch -q https://github.com/canonical/repo-policy-compliance ce8555c56b2ae28235996714849a60459515f6fc
  Running command git checkout -q ce8555c56b2ae28235996714849a60459515f6fc
  Resolved https://github.com/canonical/repo-policy-compliance to commit ce8555c56b2ae28235996714849a60459515f6fc
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Collecting flask<3.0,>=2.3
  Using cached Flask-2.3.2-py3-none-any.whl (96 kB)
Collecting flask-httpauth<5.0,>=4.8
  Using cached Flask_HTTPAuth-4.8.0-py3-none-any.whl (7.0 kB)
Collecting jsonschema<5.0,>=4.17
  Using cached jsonschema-4.17.3-py3-none-any.whl (90 kB)
Collecting pygithub<2.0,>=1.58
  Using cached PyGithub-1.58.2-py3-none-any.whl (312 kB)
Collecting pyyaml<7.0,>=6.0
  Using cached PyYAML-6.0-cp311-cp311-macosx_11_0_arm64.whl (167 kB)
Collecting Werkzeug>=2.3.3
  Using cached Werkzeug-2.3.4-py3-none-any.whl (242 kB)
Collecting Jinja2>=3.1.2
  Using cached Jinja2-3.1.2-py3-none-any.whl (133 kB)
Collecting itsdangerous>=2.1.2
  Using cached itsdangerous-2.1.2-py3-none-any.whl (15 kB)
Collecting click>=8.1.3
  Using cached click-8.1.3-py3-none-any.whl (96 kB)
Collecting blinker>=1.6.2
  Using cached blinker-1.6.2-py3-none-any.whl (13 kB)
Collecting attrs>=17.4.0
  Using cached attrs-23.1.0-py3-none-any.whl (61 kB)
Collecting pyrsistent!=0.17.0,!=0.17.1,!=0.17.2,>=0.14.0
  Using cached pyrsistent-0.19.3-cp311-cp311-macosx_10_9_universal2.whl (82 kB)
Collecting deprecated
  Using cached Deprecated-1.2.14-py2.py3-none-any.whl (9.6 kB)
Collecting pyjwt[crypto]>=2.4.0
  Using cached PyJWT-2.7.0-py3-none-any.whl (22 kB)
Collecting pynacl>=1.4.0
  Using cached PyNaCl-1.5.0-cp36-abi3-macosx_10_10_universal2.whl (349 kB)
Collecting requests>=2.14.0
  Using cached requests-2.31.0-py3-none-any.whl (62 kB)
Collecting MarkupSafe>=2.0
  Using cached MarkupSafe-2.1.3-cp311-cp311-macosx_10_9_universal2.whl (17 kB)
Collecting cryptography>=3.4.0
  Using cached cryptography-41.0.1-cp37-abi3-macosx_10_12_universal2.whl (5.3 MB)
Collecting cffi>=1.4.1
  Using cached cffi-1.15.1-cp311-cp311-macosx_11_0_arm64.whl (174 kB)
Collecting charset-normalizer<4,>=2
  Using cached charset_normalizer-3.1.0-cp311-cp311-macosx_11_0_arm64.whl (121 kB)
Collecting idna<4,>=2.5
  Using cached idna-3.4-py3-none-any.whl (61 kB)
Collecting urllib3<3,>=1.21.1
  Using cached urllib3-2.0.2-py3-none-any.whl (123 kB)
Collecting certifi>=2017.4.17
  Using cached certifi-2023.5.7-py3-none-any.whl (156 kB)
Collecting wrapt<2,>=1.10
  Using cached wrapt-1.15.0-cp311-cp311-macosx_11_0_arm64.whl (36 kB)
Collecting pycparser
  Using cached pycparser-2.21-py2.py3-none-any.whl (118 kB)
Building wheels for collected packages: repo-policy-compliance
  Building wheel for repo-policy-compliance (pyproject.toml) ... done
  Created wheel for repo-policy-compliance: filename=repo_policy_compliance-0.1.0-py3-none-any.whl size=15504 sha256=680fd8e5b7a12db5a6eb53dde7a17adb63180eb22535ea79793e3d9e237c2a2f
  Stored in directory: /Users/davidandersson/Library/Caches/pip/wheels/be/a7/1c/07401b32cd91235689681517ebfa0fa6ec26aacecd9e9d15b2
Successfully built repo-policy-compliance
Installing collected packages: wrapt, urllib3, pyyaml, pyrsistent, pyjwt, pycparser, MarkupSafe, itsdangerous, idna, click, charset-normalizer, certifi, blinker, attrs, Werkzeug, requests, jsonschema, Jinja2, deprecated, cffi, pynacl, flask, cryptography, flask-httpauth, pygithub, repo-policy-compliance
Successfully installed Jinja2-3.1.2 MarkupSafe-2.1.3 Werkzeug-2.3.4 attrs-23.1.0 blinker-1.6.2 certifi-2023.5.7 cffi-1.15.1 charset-normalizer-3.1.0 click-8.1.3 cryptography-41.0.1 deprecated-1.2.14 flask-2.3.2 flask-httpauth-4.8.0 idna-3.4 itsdangerous-2.1.2 jsonschema-4.17.3 pycparser-2.21 pygithub-1.58.2 pyjwt-2.7.0 pynacl-1.5.0 pyrsistent-0.19.3 pyyaml-6.0 repo-policy-compliance-0.1.0 requests-2.31.0 urllib3-2.0.2 wrapt-1.15.0

[notice] A new release of pip available: 22.3.1 -> 23.1.2
[notice] To update, run: pip install --upgrade pip
(venv) davidandersson@Davids-MBP-2 repo_policy_compliance % cd venv 
(venv) davidandersson@Davids-MBP-2 venv % ls
bin		include		lib		pyvenv.cfg
(venv) davidandersson@Davids-MBP-2 venv % python
Python 3.11.2 (v3.11.2:878ead1ac1, Feb  7 2023, 10:02:41) [Clang 13.0.0 (clang-1300.0.29.30)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import flask
>>> exit()
```